CONTRIBUTING.md

```diff
@@ -7,10 +7,10 @@ Getting started
 If you want to help but don't know where to start, here are some
 low-risk/isolated tasks:
 
-- [Merge a Vim patch].
 - Try a [good first issue](../../labels/good-first-issue) or [complexity:low] issue.
 - Fix bugs found by [Clang](#clang-scan-build), [PVS](#pvs-studio) or
   [Coverity](#coverity).
+- Try a [mentored project](../../labels/mentored-project).
```

Comment:
right now there is 0 issue with this label. Anyone volunteered to mentor projects ? no one could mentor someone from gsoc so that sounds overly ambitious.

Reply:
Yes, @tjdevries wanted to do this, so I created the label for him. The point was to use this for PRs (not issues), so much more low-key than a GSoC project.

Reply:
My goal is to do something like:

Add new API function skeleton `nvim_edit(name, opts)`. I don't write any of the implementation. I just put a stub where tests should go. Then someone can take the PR over and I can just help them with some of the nvim specific quirks. It is not necessarily for new programmars or for people who need hand holding along the way, but just for people who are unfamiliar w/ nvim & the way to add a new feature.